client/app/bundles/course/statistics/pages/StatisticsIndex/index.jsx

```diff
@@ -70,11 +69,7 @@ const TabPanel = (props) => {
       role="tabpanel"
       {...other}
     >
-      {value === index && (
-        <Box sx={{ pt: 3, pb: 3 }}>
-          <Typography>{children}</Typography>
```

Comment:
Actually is there any specific reason why Typography was used? Seems like the children is of JSX.Element type and if there’s div or other elements that cant be nested within p (default typography’s component), it will throw console error